app/views/admin/overdue_appointments/cancel.html.erb

```diff
@@ -0,0 +1,9 @@
+<h1>Cancel Overdue Appointment</h1>
+
+<%= form_tag('.', method: 'put') do %>
```

Comment:
Yeah you should never use relative, hard coded paths in `form_tag`. We can clean this up; see the other forms as an example

Reply:
Have an issue with this: the cancel form appears to pass `appointment_id` param instead of `id`. Get an error while rendering the cancel page:
```
No route matches {:action=>"update", :appointment_id=>"7170c3ad-01cf-44b9-9ec4-04b03e98468b", :controller=>"appointments"}, missing required keys: [:id]
```

Reply:
fixed